app.rb

```diff
@@ -1,9 +1,28 @@
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do
```

Comment:
Since the first user story is that you can see all peeps, there's no need to create a homepage here.